File: solutions/ocaml/darts/1/darts.ml
let score (x: float) (y: float): int =
  let r = sqrt (x *. x +. y *. y) in
  if r <= 1.0 then 10
  else if r <= 5.0 then 5
  else if r <= 10.0 then 1
  else 0

File: solutions/clojure/darts/1/src/darts.clj
(ns darts)

(defn score
  "Calculates the score of a dart throw."
  [x y]
  (let [unroot-radi (+ (* x x) (* y y))]
    (cond
      (<= unroot-radi 1) 10
      (<= unroot-radi (* 5 5)) 5
      (<= unroot-radi (* 10 10)) 1
      :else 0))
  )
